aireview/justfile

```diff
@@ -15,4 +13,47 @@ run:
     poetry run python src/aireview/main.py
 
 tests:
-    poetry run pytest
\ No newline at end of file
+    poetry run pytest
+
+prefect-login:
```

Comment:
**naming**: Variable $_prefect_dir is used but not defined. Define this variable at the top of the justfile or document its expected source.